R/agg_scale.R

```diff
@@ -516,7 +516,7 @@ assert_agg_scale_args <- function(dt,
 
     }
   } else {
-    assertthat::assert_that(assertive::is_null(mapping),
+    assertthat::assert_that(is.null(mapping),
```

Comment:
For later: I don't think `assertthat` buys us anything over `checkmate` and we should also consider switching that over as well